bip-0141.mediawiki

```diff
@@ -116,6 +116,16 @@ Blocks are currently limited to 1,000,000 bytes (1MB) total size. We change this
 
 The new rule is ''block weight'' ≤ 4,000,000.
 
+===== Transaction size calculations =====
+
```

Comment:
"Though not neccessarily normative, the authors suggest use of the following terms for describing transaction size", or something like that, maybe?